app/src/main/java/com/sopt/now/presentation/auth/login/LoginViewModel.kt

```diff
@@ -0,0 +1,43 @@
+package com.sopt.now.presentation.auth.login
+
+import android.os.Build.VERSION_CODES.N
+import android.util.Log
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.sopt.now.model.login.RequestLoginDto
+import com.sopt.now.model.login.ResponseLoginDto
+import com.sopt.now.utils.ApiFactory
+import com.sopt.now.utils.ErrorResponse
+import com.sopt.now.utils.NetworkUtil
+import com.sopt.now.utils.ServicePool.loginService
+import kotlinx.coroutines.launch
+
+class LoginViewModel : ViewModel() {
+    private var _status = MutableLiveData<Boolean>()
+    var status: MutableLiveData<Boolean> = _status
+
+    private var memberId: String? = null
+    fun getMemberId() = memberId
+
+    private var errorMessage: String? = null
+    fun getErrorMessage() = errorMessage
```

Comment:
요런 것들도 추후에 Uistate를 활용하게 된다면 더욱 효율적일 거에요!

Uistate와 별개로
```suggestion
    private var _memberId: String? = null
    val memberId:String? get() = _memberId

    private var _errorMessage: String? = null
    val getErrorMessage:String? get() = _errorMessage
```

이렇게 사용해도 됩니다!